components/AppNavbar.vue

```diff
@@ -27,6 +27,14 @@ const handleNavigateSend = () => {
   }
 }
 
+const handleNavigationDiscovery = () => {
+  try {
+    window.open(BASE_UP_CLOUD_URL, '_self')
+  } catch (error) {
+    console.error(error)
+  }
```

Comment:
Is this try/catch useful here? I think there is no possibility that `window.open` throw an error.